airbyte-integrations/connectors/destination-snowflake/README.md

```diff
@@ -21,15 +21,18 @@
 ```
 
 ## For Airbyte employees
-Put the contents of the `Snowflake Integration Test Config` secret on LastPass under the `Engineering` folder into `secrets/config.json` to be able to run integration tests locally.
-
-1. Put the contents of the `destination snowflake - insert test creds` LastPass secret into `secrets/insert_config.json`.
-1. Put the contents of the `destination snowflake - insert staging test creds` secret into `internal_staging_config.json`.
-1. Put the contents of the `destination snowflake - gcs copy test creds` secret into `secrets/copy_gcs_config.json`
-1. Put the contents of the `destination snowflake - s3 copy test creds` secret into `secrets/copy_s3_config.json`
-1. Put the contents of the `destination snowflake - s3 copy encrypted test creds` secret into `secrets/copy_s3_encrypted_config.json`
-1. Put the contents of the `destination snowflake - azure blob copy test creds` secret into `secrets/copy_azure_blob_config.json`
-1. Put the contents of the `destination snowflake - s3 copy encrypted test creds` secret into `secrets/copy_s3_encrypted_config.json`
+
+Put the contents of the following LastPass secrets into corresponding files under the `secrets` directory:
+
+| LastPass Secret | File |
+| --- | --- |
+| `destination snowflake - test creds (secrets/config.json)` | `secrets/config.json` |
+| `destination snowflake - insert test creds (secrets/insert_config.json)` | `secrets/insert_config.json` |
+| `destination snowflake - insert internal staging test creds (secrets/internal_staging_config.json)` | `secrets/internal_staging_config.json` |
+| `destination snowflake - s3 copy test creds (secrets/s3_copy_config.json)` | `secrets/s3_copy_config.json` |
+| `destination snowflake - s3 copy encrypted test creds (secrets/copy_s3_encrypted_config.json)` | `secrets/copy_s3_encrypted_config.json` |
+| `destination snowflake - gcs copy test creds (secrets/gcs_copy_config.json)` | `secrets/gcs_copy_config.json` |
```

Comment:
should be copy_gcs_config.json

Reply:
Actually the `copy` keyword here does not provide useful information. I will rename it to `staging`.

Reply:
Nice catch. Also actually there are two more configs...